x-pack/plugins/uptime/public/components/overview/alerts/toggle_alert_flyout_button.tsx

```diff
@@ -67,7 +67,7 @@ export const ToggleAlertFlyoutButtonComponent: React.FC<Props> = ({
       >
         <FormattedMessage
           id="xpack.uptime.navigateToAlertingButton.content"
-          defaultMessage="Manage alerts"
+          defaultMessage="Manage rules"
```

Comment:
i think potentially you will have remove existing translations from existing translated labels.

Reply:
I think you're right. You might need to manually remove them from the JSON files. Not sure if running `node scripts/i18n_check --fix` will pick this up.

Reply:
ok, thank you, I will try to fix this.

Reply:
`node scripts/i18n_check --fix` should fix any removed ids. our extraction tools automatically detect changed messages for retranslation so devs dont ahve to worry about it.